packages/react-charting/src/components/Legends/Legends.base.tsx

```diff
@@ -257,36 +236,14 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
     const overflowString = overflowText ? overflowText : 'more';
     // execute similar to "_onClick" and "_onLeave" logic at HoverCard onCardHide event
     const onHoverCardHideHandler = () => {
-      const { canSelectMultipleLegends = false } = this.props;
-      const selectedOverflowItem = find(
-        legends,
-        (legend: ILegend) =>
-          legend.title === this.state.selecetedLegendInHoverCard || legend.title === this.state.selectedLegend,
-      );
-      this.setState(
-        {
-          isHoverCardVisible: false,
-          selecetedLegendInHoverCard: 'none',
-          selectedLegends: [],
-        },
-        () => {
-          if (selectedOverflowItem) {
-            this.setState({ selectedLegend: 'none', selectedState: false }, () => {
-              if (selectedOverflowItem.action && !canSelectMultipleLegends) {
-                selectedOverflowItem.action();
-              }
-              if (this.props.onLegendHoverCardLeave && canSelectMultipleLegends) {
-                this.props.onLegendHoverCardLeave();
-              }
-              this.setState({ hoverState: false }, () => {
-                if (selectedOverflowItem.onMouseOutAction) {
-                  selectedOverflowItem.onMouseOutAction(true);
-                }
-              });
-            });
-          }
-        },
-      );
+      this.setState({ isHoverCardVisible: false });
+      const activeOverflowItem = find(legends, (legend: ILegend) => legend.title === this.state.activeLegend);
```

Comment:
Is canSelectMultipleLegends not needed in the new logic

Reply:
It is needed here:
![image](https://user-images.githubusercontent.com/110246001/197486704-71c18261-c066-4bd3-a7b4-4859faa033ef.png)

But it is not required in the onHoverCardHideHandler function